src/openforms/templates/maykin_2fa/login.html

```diff
@@ -1,11 +1,14 @@
 {% extends "maykin_2fa/login.html" %}
-{% load solo_tags i18n %}
+{% load solo_tags i18n static %}
 
 {% block extra_login_options %}
     {% get_solo 'mozilla_django_oidc_db.OpenIDConnectConfig' as oidc_config %}
     {% if oidc_config.enabled %}
-        <div class="admin-login-option">{% trans "or" %}</div>
-        <div class="admin-login-option">
+        <div class="admin-login-option admin-login-password">
```

Comment:
```suggestion
        <div class="admin-login-option admin-login-option--password">
```
would be the proper BEM approach - adding the `password` modifier lets you target it properly and it makes clear it's just a variant of a login option